### ISSUE TITLE
Excellent, but export doesn't work

### ISSUE THREAD

**Issue by MadaboutDana** [screenisland/kanban-kit#15]
I've been trying to backup the task database by exporting (to the single option, JSON). But no luck so far, despite restarts/reboots. Am I missing something?

Otherwise, love it! When's the iOS version coming out ;-)

Cheers!

**Comment by gobijan** [comment]
Hey, I'm glad you like Kanban Kit :)
What OS X Version are you using?
On my machines export works flawless. Could it be that you just updated to 10.10.3?
I will try it on a yosemite machine tomorrow and will see if I can reproduce your error.

Cheers Bijan
PS: iOS Version might be interesting as soon as I add iCloud Support (which is on my own kanban kit todo board ;)

**Comment by marksteinberg** [comment]
+1
Can’t wait!

Mark

**Comment by MadaboutDana** [comment]
Yup, that's right, it's 10.10.3. I've tried various things, but... WAIT, hold the press! I didn't try a simple change of target folder/directory! Okay, okay, it works, provided I don't use the default 'Documents' target. Well, that suits me, in fact. Brilliant! Dunno why it doesn't want to save to 'Documents', but there you go. I use a separate synced folder for most backups in any case.

So panic over! And yes, an iOS version would be VERY interesting.

**Comment by MadaboutDana** [comment]
I tell you the other thing that would be really, really cool. Some kind of support for a markup language (like the basic markup you're already using for subtasks). Just the essentials, such as **strong**/_italic_/underlined (why does nobody do underlined? Doh!), would be outstanding! Oh, and maybe a cross-page line like this: ---- or rather

---

**Comment by MadaboutDana** [comment]
Of course Markdown would be SERIOUSLY cool, and mean it was compatible with all my favourite Mac editors (Ulysses, Katana, Letterspace...). Pretty please?

**Comment by gobijan** [comment]
Markdown support might in fact be nice. Hmm...
